shared/constants/fs.js

```diff
@@ -132,3 +134,8 @@ export const makeDownloadKey = (path: Types.Path, localPath: string) =>
 
 export const downloadFilePathFromPath = (p: Types.Path): Promise<Types.LocalPath> =>
   downloadFilePath(Types.getPathName(p))
+
+export const openPathInFinderPromise = (p: Types.LocalPath): Promise<void> =>
```

Comment:
I think the plan was to pull useful pieces from kbfs saga into the fs saga, and eventually retire the old one. But for this I just added a temporary-ish thing since I knew @jzila was working on opening in Finder (in KBFS mount) and didn't want to conflict with his code. I'll remove or refactor this a little bit after https://github.com/keybase/client/pull/10757

Reply:
(@chrisnojima please correct me if I'm wrong about the kbfsSaga->fsSaga transition)

Reply:
Yeah @songgao you'll want to rebase this on keybase/client#10757, and use that API instead.

Reply:
Cool; will do after it's merged!